### PR TITLE
ENG-336 - Add AI League to teacher dashboard & move navigation to left

### PR DESCRIPTION
**Pull request by adamkecskes**
AI League is added to Teacher Dashboard. There are missing parts, but at least is is a start. 
Teacher Dashboard navigation was moved to the left. It is also collapsable. 
td-league

### CODE REVIEW

File: app/core/Router.js [hunk 1]
```diff
@@ -406,6 +406,8 @@ module.exports = (CocoRouter = (function () {
         privacy: go('PrivacyView'),
 
         'professional-development': go('core/SingletonAppVueComponentView'),
+        'ai-league': go('core/SingletonAppVueComponentView'),
```
Comment (mrfinch) [inline app/core/Router.js:409]: why this?

File: app/views/ai-league/PageLeagueTeachers.vue [hunk 2]
```diff
@@ -0,0 +1,417 @@
...
+    },
+
+    isTeacher () {
+      return me.isTeacher()
+    },
+  },
+}
+</script>
+
+<template>
```
Comment (mrfinch) [inline app/views/ai-league/PageLeagueTeachers.vue:183]: I don't prefer such large files. This should be divided into small components

File: app/views/ai-league/PageLeagueTeachers.vue [hunk 3]
```diff
@@ -0,0 +1,417 @@
...
+        <a
+          class="resources__item screen"
+          href="https://docs.google.com/presentation/d/1ouDOu2k-pOxkWswUKuik7CbrUCkYXF7N_jNjGO0II6o/edit#slide=id.gb06b5c7fa4_0_10"
+          target="_blank"
+        >
+          {{ $t('league.teacher_getting_started_guide') }}
+        </a>
+        <a
+          class="resources__item view-exemplar"
+          href="https://codecombat.com/play/ladder/storm-siege"
```
Comment (mrfinch) [inline app/views/ai-league/PageLeagueTeachers.vue:228]: we should show the current one instead of hardcoding. @smallst can you help Adam here?

Comment (smallst) [reply app/views/ai-league/PageLeagueTeachers.vue:228]: oh sure. in app core utils we have activeArenas and related function which can point to current active ai-league.

**Comment by nwinter** [comment]
Let's also migrate the /league/ladders tournament list & new tournament creation stuff into this page, too, so that the teachers can manage tournaments from inside their dashboard.

**Comment by smallst** [comment]
btw @adamkecskes i think i'm not clear on yesterday's meeting. do you know how to migrate /league/ladders tournament/new tournament creation to teacher dashboard? do you need me to help suggestions or commits?

**Comment by adamkecskes** [comment]
@smallst sure! I don't really know how/what to do. your suggestions would be highly appreciated!! :)

**Comment by adamkecskes** [comment]
@mrfinch I've pushed the modifications of teachers/ai-league page here, pls review if you have time. the arrows are still missing.

**Pull request merged**